Wrappers/Python/setup.py

```diff
@@ -16,12 +16,26 @@
 from distutils.core import setup
 import os
 import sys
+import subprocess
 
-cil_version = "21.0.0"
-if os.environ.get('CONDA_BUILD', None) is not None:
+
+cil_version = subprocess.check_output('git describe', shell=True).decode("utf-8").rstrip()
+
+
+if os.environ.get('CONDA_BUILD', 0) == 0:
+    cwd = os.getcwd()
     requires = []
 else:
     requires = ['numpy','vtk']
+    cwd = os.path.join(os.environ.get('RECIPE_DIR'),'..')
+
+fname = os.path.join(cwd, 'ccpi', 'viewer', 'version.py')
+
+if os.path.exists(fname):
+    os.remove(fname)
+with open(fname, 'w') as f:
+    f.write('version = \'{}\''.format(cil_version))
```

Comment:
Writes out the variable as 'version' but then our import lines have 'dversion' so it breaks